packages/flutter_tools/lib/src/commands/screenshot.dart

```diff
@@ -29,14 +27,7 @@ class ScreenshotCommand extends FlutterCommand {
       _kSkia,
       valueHelp: 'port',
       help: 'Retrieve the last frame rendered by a Flutter app as a Skia picture\n'
-        'using the specified diagnostic server port.\n'
-        'To find the diagnostic server port number, use "flutter run --verbose"\n'
-        'and look for "Diagnostic server listening on" in the output.'
-    );
-    argParser.addOption(
-      _kSkiaServe,
-      valueHelp: 'url',
-      help: 'Post the picture to a skiaserve debugger at this URL.',
+        'using the specified observatory port.\n'
```

Comment:
we should probably keep the help information about how to obtain the observatory port

Reply:
(suitably updated of course)

Reply:
done